### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by campbellkearns**
- adds navbar for md+ breakpoints
- adds nested mobile navbar
- made one change to font defaults in Tailwind config for pixel precision

I'm glad we chose Tailwind, this was my first time using it. I'm open to all changes.

closes #16

### CODE REVIEW

**Comment by saihaj** [comment]
image

**Comment by campbellkearns** [comment]
whoops! cool didn't realize we added these linter checks - very helpful

**Comment by campbellkearns** [comment]
also, the navbar definitely needs to be distilled into more reusable components but for now everything is there in that one component

**Review by saihaj — CHANGES_REQUESTED** [review]
great work! For now should be fine but we can refactor just create an issue so we have it there. Hover color on nav items is little hard to tell so need to change it.

File: src/components/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState<boolean>(false)
+
+  return (
+    <nav>
+      <section className="lg:container lg:mx-auto lg:px-20 xl:px-28 px-4 sm:px-8 md:px-18 my-5">
+        <section className="flex justify-between items-center">
+          <section>
+            <CustomLink href="/">
+              <Image
+                src="/static/logo_footer.png"
```
[the remark this answers is not in the document]
Comment (saihaj) [reply src/components/navbar.tsx:19]: @shubham15014

**Comment by saihaj** [comment]
test

File: src/components/navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import React, { Fragment, useState } from 'react'
+import { Dialog, Menu, Popover, Transition } from '@headlessui/react'
+import { ChevronDownIcon, MenuIcon, XIcon } from '@heroicons/react/solid'
+import Image from 'next/image'
+import CustomLink from './customLink'
+
+import { HEADER_NAV_LIST } from '../data/constants'
+
+function Nav() {
```
[the remark this answers is not in the document]
Comment (campbellkearns) [reply src/components/navbar.tsx:9]: Cool, pushing this export change now. Tomorrow I'll add custom colors to the config 🚀

**Review by saihaj — APPROVED** [review]
🚀

**Pull request merged**